ONSdigital/ras-rm-mock-eq
# update dependencies

# What and why?
This PR resolves a bunch of dependabot alerts by upgrading dependencies.

# How to test?
Do a `pipenv install`, then  `pipenv graph`, and check that the following dependency versions have been installed:

- setuptools (>=70.0.0)
- werkzeug (>=3.0.3)
- gunicorn (>=22.0.0)
- cryptography (>=43.0.1)
- urllib3 (>=2.2.2)
- requests (>=2.32.0)
- jinja2 (>=3.1.4)
- idna (>=3.7)
- black (>=24.3.0)
- certifii (>=2024.07.04)

Deploy, and ensure mock EQ service still works.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1304)
